rest/pom.xml

```diff
@@ -32,6 +32,12 @@
     <version>0.0.1-SNAPSHOT</version>
     <packaging>pom</packaging>
 
+    <properties>
+        <rest.server.url>https://ion.dod.mil/heartbeat</rest.server.url>
```

Comment:
What is this getting used for?

Reply:
Inside the spec, we have to specify a (or more) server urls that are used as examples

Reply:
Can we use something else than `.dod.mil`?